### DISCUSSION TITLE
Verfügbarkeit für Iobroker

### DISCUSSION THREAD

**Discussion by Malhiermalda** [MichaelOE/home-assistant-MyFisker#9] [Q&A] [answered]
Hey 
Bin nicht so fit mit der Materie 
Hab raspi mit ioBroker am laufen und würde gerne die hier schon möglichen Sachen gerne auch im ioBroker sehen und nutzen können 
Gibt es eine funktionierende Lösung oder Adapter der die entsprechenden Datenpunkte dort ermöglicht

**Comment by MichaelOE** [comment, marked as answer]
Hi.
sorry - but I have no knowledge of ioBroker.
I can, however, see that there is a part for ioBroker that seems to integrate Home Assistant into ioBroker.
Other than that I wont' be to much help, as ioBroker is written in JavaScript, and Home Assistant is written in python.
